tests/README.md

```diff
@@ -0,0 +1,94 @@
+
+Leverage these helpers to create your unittest as follows:
+
+```python
+import pytest
+
+from tests.base import VisualTestCase
+from yellowbrick.datasets import load_occupancy
+
+
+class MyVisualizerTests(VisualTestCase):
+
+    def test_my_visualizer(self):
+        """
+        Test MyVisualizer on a real dataset
+        """
+        # Load the data using the Yellowbrick datasets module
+        X, y = load_occupancy()
+
+        try:
+            visualizer = MyVisualizer()
+            visualizer.fit(X)
+            visualizer.poof()
+        except Exception as e:
+            pytest.fail("my visualizer didn't work")
+```
+
+### Image Comparison Tests
+
+Writing an image-based comparison test is only a little more difficult than the simple test case presented above. We have adapted `matplotlib`'s image comparison test utility into an easy to use assert method: `self.assert_images_similar(visualizer)`
+
+The main consideration is that you must specify the “baseline” (i.e. expected) image in the `tests/baseline_images/` folder structure.
+
+For example, let's say you create your unittest in `tests/test_regressor/test_myvisualizer.py` as follows:
+
+```python
+from tests.base import VisualTestCase
+...
+    def test_my_visualizer_output(self):
+        ...
+        visualizer = MyVisualizer()
+        visualizer.fit(X)
+        visualizer.poof()
+        self.assert_images_similar(visualizer)
+```
+
+The first time this test is run, there will be no baseline image to compare against, so the test will fail. Alternatively, if you are making a correction to the existing test `test_my_visualizer_output`, and the correction modifies the resulting test image, the test may also fail to match the existing baseline image. The solution is to first run the tests, then copy the new output images to the correct subdirectory under source code revision control (with `git add`). When rerunning the tests, they should now pass!
+
+We have a helper script, `tests/images.py` to clean up and manage baseline images automatically. It is run using the ``python -m`` command to execute a module as main, and it takes as an argument the path to **your** test file. To copy the figures as above:
+
+```bash
+$ python -m tests.images tests/test_regressor/test_myvisualizer.py
+```
+
+This will move all related test images from `actual_images` to `baseline_images` on your behalf (note you'll have had to already run the tests at least once to generate the images). You can also clean up images from both actual and baseline as follows:
+
+```bash
+$ python -m tests.images -C tests/test_regressor/test_myvisualizer.py
+```
+
+This is useful particularly if you're stuck trying to get an image comparison to work. For more information on the images helper script, use `python -m tests.images --help`.
```

Comment:
I think this section on the image similarity with baseline images is especially clear and helpful, very descriptive - nice work!